tests/Notificator.Redis.test.js

```diff
@@ -0,0 +1,48 @@
+const { redisPubsub } = require("../etc/config");
+const Notificator = require("../lib/Notificator");
+const notificator = require("../lib/notificatorSingletonRedis");
+const PubSub = require("../lib/PubSub");
+const RedisDriver = require("../lib/drivers/Redis");
+
+describe("Notificator with Redis driver tests", () => {
+  // test("test placeholder", async () => {});
+  test("positive: can notify/receive messages", async () => {
+    await notificator.init();
+    setTimeout(() => {
+      notificator.notify({ test: 1 });
+    }, 100);
+    const result = await new Promise((res) =>
+      notificator.receive((msg) => res(msg))
+    );
+    expect(result.test).toBe(1);
+  });
+  // test("positive: can notify/receive messages between different nodes", async () => {
+  //   const notificatorOne = createNotificator();
+  //   const notificatorTwo = createNotificator();
+  //   await notificatorOne.init();
+  //   await notificatorTwo.init();
+  //   setTimeout(() => {
+  //     notificatorOne.notify({ test: 1 });
+  //   }, 100);
+  //   const result = await new Promise((res) =>
+  //     notificatorTwo.receive((msg) => res(msg))
+  //   );
+  //   expect(result.test).toBe(1);
+  // });
```

Comment:
Would be great to have both tests working (with multiple nodes also)

Reply:
Actually, we have two tests here: one for the Redis https://github.com/unsigned6/notificator/blob/efb3b7640e8e6a559c92748f83bd146d9c57a447/tests/Notificator.Redis.test.js and one for the Rabbithttps://github.com/unsigned6/notificator/blob/efb3b7640e8e6a559c92748f83bd146d9c57a447/tests/Notificator.Rabbit.test.js